cmd/cluster-agent/api/v1/clusterchecks.go

```diff
@@ -181,7 +181,7 @@ func shouldHandle(w http.ResponseWriter, r *http.Request, h *clusterchecks.Handl
 
 // clusterChecksDisabledHandler returns a 404 response when cluster-checks are disabled
 func clusterChecksDisabledHandler(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(http.StatusPreconditionFailed)
+	w.WriteHeader(http.StatusNotFound)
```

Comment:
The existing comment already said that this should return 404 instead of 412 :)
The change looks good to me. Could something break? Any callers that might be depending on a specific error code? (I'm not too familiar with this part of the code).

Reply:
It should be safe 👌